### PR TITLE
Set default task status to active

### PR DESCRIPTION
**Pull request by hmasila**
Set default task status to active

There are two approaches involved:
1. Having a `before_save` snippet that sets the status to 'Active' if the status is not provided
2. Adding a default value in the db migration for tasks model

### CODE REVIEW

File: app/views/tasks/_form.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <div class="form-inputs">
     <%= f.input :title %>
     <%= f.input :priority, as: :select, collection: Task::PRIORITIES %>
-    <%= f.input :status, :as => :select, :collection => ['Active', 'Cancelled'] %>
+    <%= f.input :status, :as => :select, :collection => ['Active', 'Cancelled'], :selected => 'Active' %>
```
Comment (kgf) [inline app/views/tasks/_form.html.erb]: This was much closer to what I was expecting, but it would be better to have this in the controller.

Comment (hmasila) [reply app/views/tasks/_form.html.erb]: Let me try that then

Comment (kgf) [reply app/views/tasks/_form.html.erb]: Thanks !

File: db/migrate/20171026191711_change_column_default.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class ChangeColumnDefault < ActiveRecord::Migration
```
Comment (kgf) [inline db/migrate/20171026191711_change_column_default.rb]: I would prefer to not put a default in the database.

**Comment by kgf** [comment]
@hmasila Thanks for your help !

**Pull request merged**